### PR TITLE
Definition for @sign

### PR DESCRIPTION
**Pull request by L1AMcB**
**- What I did**
Added the assign definition
**- How I did it**
took the definition from atsigns FAQ pages
**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Can you add some more content to this, and maybe some examples.
See [this video](https://vimeo.com/718810635) as a reference

**Review by Jesseadriano10 — CHANGES_REQUESTED** [review]
Looks good so far! A few suggestions would be to add keywords such as CRAM, PKAM, Secondary server the link that redirects to its proper definition page.

**Comment by Jesseadriano10** [comment]
> Can you add some more content to this, and maybe some examples. See [this video](https://vimeo.com/718810635) as a reference

Also check out this section of Atsign.com => https://atsign.com/what-is-an-sign/

**Comment by L1AMcB** [comment]
I'm not entirely sure about the example so let me know what you guys think!

**Review by JeremyTubongbanua — DISMISSED** [review]
LGTM

File: content/docs/reference/atsign.md [hunk 1]
```diff
@@ -13,4 +13,19 @@ toc: false # Change this to "true" to show the table of contents
 weight: 200 # For single pages, lower is first.
 ---
 
-TODO @sign definition goes here
+## What is an @sign
+An @sign is a handle (i.e. @alice) that functions as your digital identity. It uses end-to-end encryption to ensure that your data is 100% owned and controlled by you.
```
Comment (xavierchanth) [inline content/docs/reference/atsign.md]: `i.e.` > `e.g.`

**Review by xavierchanth — DISMISSED** [review]
Just the one change.
`i.e.` is latin for `id est` meaning `that is` - used to further explain something.
`e.g.` is latin for `exempli gratia` meaning `for example` - used for examples.

File: content/docs/reference/atsign.md (hunk 1 above)
Comment (L1AMcB) [reply content/docs/reference/atsign.md]: I'll fix it! I took that definition from the atsign FAQ's and it shows "i.e." in the Assign FAQ's so it needs to be fixed there as well

**Review by Jesseadriano10 — APPROVED** [review]
Looks good to me!

**Pull request merged**